gnd/src/main/java/com/google/android/gnd/repository/local/Converters.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.repository.local;
+
+import static java8.util.J8Arrays.stream;
+
+import androidx.room.TypeConverter;
+
+public abstract class Converters {
```

Comment:
If this is meant to be a non-instantiable utility class, I suggest making it final (not abstract) and making the constructor private to prevent instantiation and subclassing.